src/pages/logosValidator/dashboardDefinition.ts

```diff
@@ -0,0 +1,898 @@
+  },
+  "79-GL": {
+    tag: "79-GL",
+    label: "79-GL",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/79-gl.73x90.svg",
+  },
+  "80-Paper": {
+    tag: "80-Paper",
+    label: "80-Paper",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/80-paper73x90.svg",
+  },
+  "81-PapPet": {
+    tag: "81-PapPet",
+    label: "81-PapPet",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/81-pappet.73x90.svg",
+  },
+  "82": {
+    tag: "82",
+    label: "82",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/82.73x90.svg",
+  },
+  "83": {
+    tag: "83",
+    label: "83",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/83.73x90.svg",
+  },
+  "84-C/PAP": {
+    tag: "84-C/PAP",
+    label: "84-C/PAP",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/84-c-pap.73x90.svg",
+  },
+  "84-PapAl": {
+    tag: "84-PapAl",
+    label: "84-PapAl",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/84-papal.73x90.svg",
+  },
+  "85": {
+    tag: "85",
+    label: "85",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/85.73x90.svg",
+  },
+  "87-CSL": {
+    tag: "87-CSL",
+    label: "87-CSL",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/87-csl.73x90.svg",
+  },
+  "90": {
+    tag: "90",
+    label: "90",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/90.73x90.svg",
+  },
+  "91": {
+    tag: "91",
+    label: "91",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/91.73x90.svg",
+  },
+  "92": {
+    tag: "92",
+    label: "92",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/92.73x90.svg",
+  },
+  "95": {
+    tag: "95",
+    label: "95",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/95.73x90.svg",
+  },
+  "96": {
+    tag: "96",
+    label: "96",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/96.73x90.svg",
+  },
+  "97": {
+    tag: "97",
+    label: "97",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/97.73x90.svg",
+  },
+  "98": {
+    tag: "98",
+    label: "98",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/98.73x90.svg",
+  },
+  "de:einwegpfand": {
+    tag: "de:einwegpfand",
+    label: "Einwegpfand",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/de/packaging/einwegpfand.70x90.svg",
+  },
+  "en:fsc": {
+    tag: "en:fsc",
+    label: "FSC",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/fsc.70x90.svg",
+  },
+  "en:fsc-mix": {
+    tag: "en:fsc-mix",
+    label: "FSC-Mix",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/fsc.70x90.svg",
+  },
+  "en:pefc": {
+    tag: "en:pefc",
+    label: "PEFC",
+    logo: "https://world-fr.openfoodfacts.org/images/lang/en/packaging/pefc.70x90.svg",
+  },
```

Comment:
You already added those logos. An object can not have two times the same key. That's what is breaking the Netlify deployement
```suggestion
```